### DISCUSSION TITLE
schema.org Feature Requests

### DISCUSSION THREAD

**Discussion by Jezjames** [joomla/joomla-cms#42856] [2. Cold Features]
I am working on Joomla 5 and I really love it.
I really like the new schema.org plugin for SEO optimization however I  do have some requests which I hope will go in a long way in improving the product.

1.  We need the ability to edit or have the schema.org plugin in the category pages.
2. Add the following very important content types: product for tangible goods or to Give Google detailed product information such as price, availability, and review ratings and service for intangible goods offered by individuals and businesses or Mark up a service provided by an organization, e.g. delivery service, print services, etc.
3. Add the following content types if possible: Course, Fact Check, FAQ, HowTo, Local Business, Video, Place, TouristDestination, Trip, TravelAgency, OnlineBusiness, itinerary, TouristTrip

Thanks

**Comment by ManuelVoileux** [comment]
Hi, I agree with the need for other Schema.org plugin.

I have prepared plugins for FAQ, NewsArticle, VideoObject, and I have adapted the plugin for Recipe, in order to have the proper type for the author, the publisher, the copyright owner and the image.
These have been tested with the shema validator tool, https://validator.schema.org/   and the more specific Google tool : https://search.google.com/test/rich-results?hl=en
I am open to share. As I have very limited coding capabilities, these need checking.

Thanks for sharing your wishlist. 1st I will check if these are recognized by Google. Schema accepts every schema as long as the coding is OK. Google does not. I could not get Google recognize the Guide , or LearningResource Schema
